growth/too/tasks/gattini_client.py

```diff
@@ -0,0 +1,62 @@
+from astropy import time
+import astropy.units as u
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+
+from ..flask import app
+from . import celery
+from .. import models
+
+import psycopg2
+import psycopg2.extras
+
+log = get_task_logger(__name__)
+
+__all__ = ('gattini_obs')
+
+
+def get_obs(start_time, end_time):
+
+    conn = psycopg2.connect(host=app.config["GATTINI_HOST"],
+                            dbname=app.config["GATTINI_DBNAME"],
+                            user=app.config["GATTINI_USER"],
+                            password=app.config["GATTINI_PASSWORD"])
+    cur = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+
+    query = 'SELECT ss.field, ss.fieldseq, min(ss.datebeg) AS utstart, ' +\
```

Comment:
Please use a multiline string (`""" .... """`) to avoid the explicit line continuations.